chanzuckerberg/idseq-web
# Show visualizations list in new tabbed navigation

# Description

This adds a list of saved visualizations that you can use to re-open a visualization with state. 

![image](https://user-images.githubusercontent.com/28797/53277376-6b160b80-36b8-11e9-958f-ebefaf20f4ed.png)

NOTE: like the other lists in the new tabs, this list still needs final visual design. For now, only admins can save a viz, and only users with the feature flag can see the lists. It's not possible in the UI yet to make a viz public.

## Type of change


# Test and Reproduce
Outline the steps to test or reproduce the PR here.

1. open "my library"
2. see list of private saved vizs (manually assigned in rails console)
3. click link for each type of viz
4. see stateful viz

1. open "public"
2. see list of public enabled viz (manually assigned in rails console)
3. click link for each type of viz
4. see stateful viz
